app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -73,33 +73,38 @@ public function getFeed($storeid, $type = '')
      * @param $order
      * @return bool
      */
-    public function sendInvitation($order)
+    public function sendInvitation(Mage_Sales_Model_Order $order): bool
     {
-        $startTime = microtime(true);
-        $crontype = 'orderupdate';
-        $storeId = $order->getStoreId();
-        $apiId = trim(Mage::getStoreConfig('webwinkelconnect/general/api_id', $storeId));
-        $apiKey = trim(Mage::getStoreConfig('webwinkelconnect/general/api_key', $storeId));
-        $postData['email'] = $order->getCustomerEmail();
-        $postData['order'] = $order->getIncrementId();
-        $postData['delay'] = trim(Mage::getStoreConfig('webwinkelconnect/invitation/delay', $storeId));
-        $postData['customer_name'] = $order->getCustomerName();
-        $postData['client'] = 'magento';
-        $postData['language'] = $this->getLanguage($storeId, $order);
+        $start_time = microtime(true);
+        $cron_type = 'orderupdate';
+        $store_id = $order->getStoreId();
+        $helper = Mage::helper('webwinkelconnect');
+        $post_data['email'] = $order->getCustomerEmail();
+        $post_data['order'] = $order->getIncrementId();
+        $post_data['delay'] = trim(Mage::getStoreConfig('webwinkelconnect/invitation/delay', $store_id));
+        $post_data['customer_name'] = $order->getCustomerName();
+        $post_data['client'] = 'magento';
+        $post_data['language'] = $this->getLanguage($store_id, $order);
```

Comment:
Please avoid unnecessarily renaming variables together with functionality changes, it makes the diff very difficult to read.